src/server/server.ts

```diff
@@ -130,9 +212,9 @@ export class GraphQLTransportWSServer {
     return (message: any) => {
       let parsedMessage: RequestMessage;
       try {
-        parsedMessage = this.parseLegacyProtocolMessage(connectionContext, JSON.parse(message));
+        parsedMessage = GraphQLTransportWSServer.parseLegacyProtocolMessage(connectionContext, JSON.parse(message));
       } catch (e) {
-        this.sendMessage(connectionContext, null, MessageTypes.GQL_ERROR, { message: e.message });
+        GraphQLTransportWSServer.sendMessage(connectionContext, null, MessageTypes.GQL_ERROR, { message: e.message });
```

Comment:
a. Why 'this' is lost?
b. let's create sendError function which will just get context and error, because in the future we might want to attach more info to the errors from the server.